bytestring/build.gradle.kts

```diff
@@ -0,0 +1,80 @@
+import org.jetbrains.dokka.gradle.DokkaTaskPartial
+import org.jetbrains.kotlin.gradle.plugin.KotlinSourceSet
+
+plugins {
+    kotlin("multiplatform")
+    id("org.jetbrains.kotlinx.kover") version "0.7.1"
+    id("org.jetbrains.dokka") version "1.8.20"
+}
+
+kotlin {
+    jvm {
+        withJava()
+        testRuns["test"].executionTask.configure {
+            useJUnitPlatform()
+        }
+    }
+
+    js(IR) {
+        nodejs {
+            testTask {
+                useMocha {
+                    timeout = "30s"
+                }
+            }
+        }
+        browser {
+            testTask {
+                filter.setExcludePatterns("*SmokeFileTest*")
+                useMocha {
+                    timeout = "30s"
+                }
+            }
+        }
+    }
+
+    configureNativePlatforms()
+    sourceSets {
+        val commonMain by getting
+        val commonTest by getting {
+            dependencies {
+                implementation(kotlin("test"))
+            }
+        }
+        val jvmMain by getting
+        val jvmTest by getting
+
+        createSourceSet("nativeMain", parent = commonMain, children = nativeTargets)
+        createSourceSet("nativeTest", parent = commonTest, children = nativeTargets)
+    }
+
+    explicitApi()
+    sourceSets.configureEach {
+        configureSourceSet()
+    }
+}
+
+fun KotlinSourceSet.configureSourceSet() {
+    val srcDir = if (name.endsWith("Main")) "src" else "test"
+    val platform = name.dropLast(4)
+    kotlin.srcDir("$platform/$srcDir")
+    if (name == "jvmMain") {
+        resources.srcDir("$platform/resources")
+    } else if (name == "jvmTest") {
+        resources.srcDir("$platform/test-resources")
+    }
+    languageSettings {
+        progressiveMode = true
+    }
+}
+
+tasks.withType<DokkaTaskPartial>().configureEach {
+    dokkaSourceSets.configureEach {
+        includes.from("Module.md")
+
+        perPackageOption {
+            suppress.set(true)
+            matchingRegex.set(".*unsafe.*")
```

Comment:
Do you think it's worth excluding it from the official documentation?

Reply:
Otherwise, it looks like a Forbidden fruit - we're asking not to use it by any means and then making it visible to everyone.